src/dbnode/storage/shard.go

```diff
@@ -875,6 +876,8 @@ func (s *dbShard) purgeExpiredSeries(expiredEntries []*lookup.Entry) {
 		series.Close()
 		s.list.Remove(elem)
 		s.lookup.Delete(id)
+		s.logger.Info("entry expired",
```

Comment:
I've added this line because it helped me to catch entry expiration bug. I guess it could be removed or should we change to debug log level here (might be useful to have logging here for future troubleshooting)?

Reply:
Not sure how common this message would be. In any case, it feels like `debug` level to me at least.

Reply:
removed.